bot.py

```diff
@@ -0,0 +1,24 @@
+"""
+Awesomebot entry point
+Mike Tung (seekheart)
+"""
+
+import discord
+import asyncio
+from settings import CREDS, PREFIX, logger
+
+
+
+def init_bot():
+    """Creates a discord client connection"""
+    return discord.Client()
+
```

Comment:
Should we be using the `discord.bot.ext` frameworks? Also this feels like an unpythonic function.

Reply:
not sure what that is